R/LearnerSurvRpart.R

```diff
@@ -78,12 +78,8 @@ LearnerSurvRpart = R6Class("LearnerSurvRpart",
     },
 
     .predict = function(task) {
-      PredictionSurv$new(
-        task = task,
-        crank = invoke(predict,
-          object = self$model,
-          newdata = task$data(cols = task$feature_names)
-      ))
+      preds = invoke(predict, object = self$model, newdata = task$data(cols = task$feature_names))
+      list(crank = preds)
```

Comment:
how is the task passed to the `PredictionSurv` in the new method?

Reply:
It is not used any more. I'm thinking about removing the argument from the constructor.

Reply:
How does this work for measures that require training set?

Reply:
RR and BMR have the task and indices and pass it to the measure's score function if required. A manually trained learner has a task stub stored internally which contains the training set.